src/main/java/com/yelp/nrtsearch/server/luceneserver/QueryNodeMapper.java

```diff
@@ -57,7 +57,7 @@ public Query getQuery(com.yelp.nrtsearch.server.grpc.Query query, IndexState sta
     Query queryNode = getQueryNode(query, state);
 
     if (query.getBoost() < 0) {
-      throw new IllegalArgumentException("Boost must be a positive number, query: " + query);
```

Comment:
This could throw the same errors as in https://github.com/Yelp/nrtsearch/pull/242 if the query is too large.
```
Jan 19, 2021 5:33:01 PM io.grpc.netty.shaded.io.grpc.netty.NettyServerHandler onStreamError
WARNING: Stream Error
io.grpc.netty.shaded.io.netty.handler.codec.http2.Http2Exception$HeaderListSizeException: Header size exceeded max allowed size (8192)
...
```

Reply:
oops, this is a removal. all good!